PennMobile/Setup + Navigation/RootViewController.swift

```diff
@@ -373,4 +391,24 @@ extension RootViewController : TwoFactorEnableDelegate {
             askForNotificationPermission()
         }
     }
+    ///Checks whether the user has enabled Two Factor but the code was never fetched. This can happen when the user closes the
+    ///app before it is done fetching the code. Don't try to fetch it if Two Factor was enabled more than a week ago. Also, don't try to fetch
+    ///it again if the user already tried again once.
```

Comment:
Use multi-line documentation / a block comment. See [here](https://sarunw.com/posts/swift-documentation/#syntax) for example.